tikv-client/src/main/java/com/pingcap/tikv/region/TiRegion.java

```diff
@@ -87,8 +87,7 @@ public Peer getLeader() {
   }
 
   public List<Peer> getLearnerList() {
-    int peerCnt = getMeta().getPeersCount();
-    List<Peer> peers = new ArrayList<Peer>();
+    List<Peer> peers = new ArrayList<>();
```

Comment:
`getLearnerList` is not used, maybe we should delete this function

Reply:
We use this function to get learner store.

Reply:
It is used in CoprocessorIterator.process.